folly/Subprocess.cpp

```diff
@@ -24,6 +24,7 @@
 #include <sys/prctl.h>
 #endif
 #include <fcntl.h>
+#include <unistd.h>
```

Comment:
This is already included via the `folly/portability/Unistd.h` include further down. Remove this include.